sdk/identity/identity/test/manual/src/index.tsx

```diff
@@ -85,7 +98,7 @@ function ClientDetailsEditor({ clientDetails , onSetClientDetails }: ClientDetai
 
 function useKeyVaultKeys(vaultName: string, clientDetails: ClientDetails) {
   const [running, setRunning] = React.useState(false)
-  const [keys, setKeys] = React.useState<Key[]>(undefined)
+  const [keys, setKeys] = React.useState<DeletedKey[]>(undefined)
```

Comment:
Why switch from Key to DeletedKey here? If I'm remembering right, these are only part of when you list deleted keys, but that doesn't seem to be what's happening here

Reply:
This is my mistake! I'll update it. Thank you!